lib/src/components/NxDrawer/NxDrawer.tsx

```diff
@@ -84,14 +89,16 @@ function NxDrawer(props: Props) {
     onCancel,
     children,
     variant,
+    closeDisabled,
+    closeBtnTooltip,
     ...attrs
   } = props;
 
   const [openState, setOpenState] = useState<OpenState>('closed');
 
   const dialogRef = useRef<HTMLDialogElement>(null);
 
-  const closeDrawer = () => onClose();
+  const closeDrawer = closeDisabled ? () => {} : () => onClose();
```

Comment:
Unless there are concerns over the undocumented parameters that this would pass to `onClose`
```suggestion
  const closeDrawer = closeDisabled ? () => {} : onClose;
```